src/main/java/fi/otavanopisto/kuntaapi/server/controllers/EventController.java

```diff
@@ -86,4 +100,61 @@ public AttachmentData getEventImageData(Integer size, OrganizationId organizatio
     
     return Collections.unmodifiableList(result);
   } 
+  
+  private class EventComparator implements Comparator<Event> {
+    
+    private EventOrder order;
+    private EventOrderDirection direction;
+    
+    public EventComparator(EventOrder order, EventOrderDirection direction) {
+      this.order = order;
+      this.direction = direction;
+    }
+    
+    @Override
+    public int compare(Event event1, Event event2) {
+      int result;
+      
+      switch (order) {
+        case END_DATE:
+          result = compareEndDates(event1, event2);
+        break;
+        case START_DATE:
+          result = compareStartDates(event1, event2);
+        break;
+        default:
+          result = 0;
+        break;
+      }
+      
+      if (direction == EventOrderDirection.ASCENDING) {
+        return -result;
+      } 
+      
+      return result;
+    }
+
+    private int compareStartDates(Event event1, Event event2) {
+      return compareDates(event1.getStart(), event2.getStart());
+    }
+
+    private int compareEndDates(Event event1, Event event2) {
+      return compareDates(event1.getEnd(), event2.getEnd());
+    }
+    
+    private int compareDates(OffsetDateTime dateTime1, OffsetDateTime dateTime2) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "compareDates" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)